src/charm.py

```diff
@@ -143,11 +211,18 @@ def _on_collect_status(self, event: ops.CollectStatusEvent) -> None:
 
     def _on_config_changed(self, _: ops.ConfigChangedEvent) -> None:
         """Handle changed configuration event."""
+        self.unit.status = ops.MaintenanceStatus("Configuring workload")
+        self.bind.configure(
+            {
+                "django-debug": "true" if self.config["django_debug"] else "false",
+                "django-allowed-hosts": self.config["django_allowed_hosts"],
```

Comment:
It might be worth to validate this setting

Reply:
I added a test, It needs https://github.com/canonical/charmed-bind/pull/21 to fully work